backpack/__init__.py

```diff
@@ -66,6 +74,15 @@ def __enter__(self):
         CTX.set_extension_hook(self.extension_hook)
 
     def __exit__(self, type, value, traceback):
+        """Leave backpack environment.
+
+        Args:
+            type: .
+            value: .
+            traceback: .
+        """
+        for backprop_extension in CTX.get_active_exts():
+            backprop_extension.clear()
```

Comment:
Delete and test that `saved_quantities` is empty after backpropagation with `retain_grad=False`.

Reply:
Deleted. Test still missing.

Reply:
Added the test.